spec/fixtures/state_file/fed_return_jsons/2023/md/riley_claimedasdep.json

```diff
@@ -6,7 +6,6 @@
       "middleInitial": null,
       "lastName": "Gilmore",
       "dateOfBirth": "1980-01-01",
-      "isPrimaryFiler": true
```

Comment:
Did you confirm this is accurate? I would expect this filer to still have `"isPrimaryFiler": true` on this return even if they are claimed as a dependent on another return, but I can accept I might be wrong if you've already asked about this somewhere else.